tests/schema.py

```diff
@@ -147,7 +147,7 @@ def make(self, key):
         from datetime import date, timedelta
 
         users = [None, None] + list(User().fetch()["username"])
-        random.seed("Amazing Seed")
+        random.seed("Amazing Seed4")
```

Comment:
For `test_restriction_by_null`, we needed some users in this table with name `None`. I changed the seed until the table contents fulfilled this restriction, and `test_restriction_by_null` passed.